src/lib/hooks/use-best-quote/index.ts

```diff
@@ -228,6 +229,7 @@ async function getFlashMintQuote(
   const gasStation = new GasStation(provider)
   const gasPrice = await gasStation.getGasPrice()
 
+  // TODO: use timestamps to check how long the loop's been running
   while (true) {
```

Comment:
thinking through this - why is the loop needed here? (wondering if there's a way to avoid the loop + need to add timestamps)

Reply:
the loop right now is the way of approximation of the best input amount quote (when minting). so far I never came up with something else of how to approximate - unless you fetch a FM quote at least once.

Reply:
Let's continue the discussion online. Updates will be made in new PR. @0xonramp